setup.py

```diff
@@ -26,7 +26,7 @@ def package_data(pkg, roots):
 
 setup(
     name="openedx-scorm-xblock",
-    version="15.0.0",
+    version="15.1.0",
```

Comment:
This PR introduces a backward-compatible change, so we should only bump to 15.0.1 https://docs.tutor.overhang.io/tutor.html#versioning